### ISSUE TITLE
Update figures in Episode 3 to have common y axis scales for easier comparison

### ISSUE THREAD

**Issue by amithjkamath** [ubern-mia/bender#1]
See https://www.linkedin.com/feed/update/urn:li:activity:6975742407854198784?commentUrn=urn%3Ali%3Acomment%3A%28activity%3A6975742407854198784%2C6975900964708888576%29

"It would help to see the left and right plots with the same y axis min/max values. I think even without doing that, the green curve has the most consistency between the training and validation. I'd be interested to see how a subsequent test set performs!" - for the comparison curves in Episode 3, where we look at the training and validation losses and accuracies - having the same y scale would help better compare the two. Update the figures so this is clearer next.
